.github/workflows/automerge.yml

```diff
@@ -10,34 +10,67 @@ jobs:
     name: Merge Master to Staging
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set Git config
         run: |
           git config --local user.email "Sofie-Automation@users.noreply.github.com"
           git config --local user.name "GitHub Actions # Sofie-Automation"
       - name: Merge 'master' to 'staging'
+        id: Merge
+        continue-on-error: true
         run: |
           git fetch --unshallow
           git checkout staging
           git pull
           git merge --no-ff origin/master -m "Automerge 'master' to 'staging'"
           git push
+      - name: Abort Merge
+        if: steps.Merge.outcome == 'failure'
+        run: |
+          echo "Merge was a ${{steps.Merge.outcome}}."
```

Comment:
The echo is unnecessary. You have already specified that outcome is `failure` if this i executed. Same for next job.